src/messages/serenity.proto

```diff
@@ -49,8 +49,6 @@ message Contention{
     NETWORK = 4;
   }
 
-  optional WorkID victim = 1;
```

Comment:
Why did you remove victim? Is it not needed any more? What if we have multiple PR tasks on node and want to assure QoS for all of them?